cogs/utils/auto_incorrect.py

```diff
@@ -0,0 +1,73 @@
+import sys, random
+
+
+# Credit to lazho for this function
+def auto_incorrect(input_str):
+    chance = 99
+    input_str = input_str.split()
+
+    specialCases = {
```

Comment:
Just a minor style change: can you use all lowercases and separate words by underscores for the 2 variables `specialCases` and `outputStr`?

https://www.python.org/dev/peps/pep-0008/#function-and-variable-names